src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -306,6 +306,7 @@ internal static int GetWarningLevel(ErrorCode code)
                 case ErrorCode.WRN_CallerLineNumberPreferredOverCallerMemberName:
                 case ErrorCode.WRN_CallerLineNumberPreferredOverCallerFilePath:
                 case ErrorCode.WRN_DelaySignButNoKey:
+                case ErrorCode.WRN_AttributeIgnoredWhenPublicSigning:
```

Comment:
There are other warnings that I don't see in this list. I don't think the warning level is currently used for anything in the compiler.

---
In reply to: [63420965](https://github.com/dotnet/roslyn/pull/11334#discussion_r63420965) [](ancestors = 63420965)